samples/emoji.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see license.html or https://sdk.ckeditor.com/license.html.
+-->
+<html>
+	<head>
+		<meta charset="utf-8">
+		<meta name="description" content="Emoji plugin">
+		<meta name="keywords" content="ckeditor, editor, wysiwyg, emoji">
+		<meta name="sdk-samples" content="Default emoji configuration|Customised emoji configuration">
+		<title>Emoji plugin</title>
+		<link href="https://fonts.googleapis.com/css?family=Maven+Pro:700,500,400" rel="stylesheet">
+		<link href="../template/theme/css/sdk.css" rel="stylesheet">
+		<script src="../../ckeditor-dev/ckeditor.js"></script>
+		<script src="assets/picoModal-2.0.1.min.js"></script>
+		<script src="assets/sample.js"></script>
+		<script src="assets/contentloaded.js"></script>
+		<script src="assets/simplesample.js"></script>
+		<script src="assets/beautify-html.js"></script>
+		<!--[if lt IE 9]>
+		<script src="assets/html5shiv.min.js"></script>
+		<![endif]-->
+	</head>
+	<body>
+		<header class="sdk-header">
+			</header>
+			<section class="sdk-container">
+				<nav class="sdk-sidebar">
+				</nav>
+
+				<section class="sdk-contents">
+					<h1>Emoji plugin</h1>
+					<p>Emoji plugin provides possibility to easily insert utf symbols, commonly known as emoji, inside editor content.
+						Every emoji has it's own name which is used to select proper symbol from available set. By default pressing <code>:</code> activates drop down suggestion box
+						where user can select proper emoji. User can type emoji name after <code>:</code> character to narrow down suggestions on the list.</p>
+					<p>Plugin provides configuration options.</p>
```

Comment:
SDK sample is not a proper place to enumerate config properties. This should be done in dev docs.